scripts/exrrfs_ungrib.sh

```diff
@@ -66,6 +66,11 @@ for fhr in  ${fhr_all}; do
     NAME_FILE=${NAME_PATTERN/fHHH/${HHH}}
     GRIBFILE="${COMIN}/${NAME_FILE}"
     ${cpreq} ${GRIBFILE} ${GRIBFILE_LOCAL}
+    if [[ ! -z "${NAME_PATTERN_b}" ]]; then
```

Comment:
Besides GFS and GEFS, do we have any other situations where two grib2 files are to be combined?

Reply:
I think GFS and GEFS are the only two model source we might need to combine the "a" and "b" files. I talked with @hu5970 and right now "GFS" and "GEFS" are reserved to the operational usage on WCOSS. Therefore if we use GFS or GEFS data on Jet or Hera, and if we don't have the merged files ready, we need to specify the two files and cat them under "else". We do need to figure out how to best handle them later.

Reply:
We don't need separate logic for wcoss2 and jet. 
We have operational GFS data on Jet at:  `/public/data/ops/NOAA/NODD` which is maintained by the GSL DSG team.
We will have operational GEFS data as well soon.

Reply:
"GFS" and "GEFS" is reserved for operational environmental runs (WCOSS2 real-time run) only. All others runs need to set up the path and file name pattern.

Reply:
> "GFS" and "GEFS" is reserved for operational environmental runs (WCOSS2 real-time run) only. All others runs need to set up the path and file name pattern.

Why is it so? I don't get this.
For GFS/GEFS, we can use operational products on any machines including Jet. Why would we use different logic for wcoss2 and RDHPCS? (1) GSL DSG fetches real-time gribA/B GFS/GEFS from AWS  (2) It is also much easier to stage gribA/B GFS/GEFS from AWS or HPSS (3) We better keep a consistent interface for users.

@chunhuazhou If you want to use the name pattern, let's use the pre-combined GEFS file to avoid making the workflow unnecessarily complicated.

Reply:
@chunhuazhou Let's make the logic simple. If we want to use the "name pattern", let's use the combined GFS and GEFS files and hence there is no need to modify all those Python files.